src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -138,8 +138,35 @@ void RotatedSPOs::apply_rotation(const std::vector<RealType>& param, bool use_st
     rot_mat[p][q] = -x;
   }
 
+  // JPT DEBUG
+  app_log() << "Kappa:\n";
+  app_log() << " " << rot_mat.size1() << " x " << rot_mat.size2() << "\n";
+  app_log() << std::fixed;
+  for (auto i = 0; i < rot_mat.size1(); i++)
+  {
+    for (auto j = 0; j < rot_mat.size2(); j++)
+    {
+      app_log() << " " << std::setw(10) << std::setprecision(5) << rot_mat[i][j];
+    }
+    app_log() << "\n";
+  }
+
   exponentiate_antisym_matrix(rot_mat);
 
+  // JPT DEBUG
```

Comment:
Please remove debug printout.

Reply:
The debug output can be put under an if statement with a variable that is off by default.
That's better than an `#ifdef` or commenting out the debug output, because then it gets kept up to date.

Reply:
Removed. The rotation parameters are output to app_log() if they are included in the optimization, so in principle nothing is lost by removing this.